data/data/agent/systemd/units/agent-copy-iso-files.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Copy the additional files stored in the agent ISO during the boot
+Wants=network-online.target
+After=network-online.target
```

Comment:
Is there a reason it needs to wait for network-online? Aren't we just copying files on the file system and not from the network?

Reply:
Not really, just wanted to be sure to be "late" enough in the [boot sequence](https://www.freedesktop.org/software/systemd/man/systemd.special.html), so I guess that in this caso also `basic.target` would work. Do you have any specific suggestion?

Reply:
I'm wondering how it will interact with the agent-interactive-console.service dependencies: 
```
After=network-pre.target NetworkManager.service pre-network-manager-config.service selinux.service
Before=network.target network.service agent.service NetworkManager-wait-online.service
```

Reply:
It seems a fair point, thanks @rwsu. Since it's a service essentially required for making work `agent-interactive-console.service` it seems a good point at least to start them together.